translate/src/modules/translationform/components/EditAccesskey.tsx

```diff
@@ -1,68 +1,145 @@
-import React, { useContext } from 'react';
+import React, {
+  forwardRef,
+  memo,
+  useCallback,
+  useContext,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from 'react';
 
-import { EditorActions, useEditorValue } from '~/context/Editor';
+import { EditFieldHandle, EditorActions, EditorData } from '~/context/Editor';
 import { Locale } from '~/context/Locale';
-import { useHandleShortcuts } from '~/modules/editor';
 import { useReadonlyEditor } from '~/hooks/useReadonlyEditor';
+import { useCopyOriginalIntoEditor } from '~/modules/editor';
 import { extractAccessKeyCandidates } from '~/utils/message';
 
+import { useHandleEnter, useHandleEscape } from '../utils/editFieldShortcuts';
 import type { EditFieldProps } from './EditField';
 
-export function EditAccesskey({
-  id,
-  inputRef,
-  name,
-  onFocus,
-  userInput,
-  value,
-}: EditFieldProps & { name: string }) {
-  const locale = useContext(Locale);
-  const { setEditorFromInput } = useContext(EditorActions);
-  const message = useEditorValue();
+function useHandleShortcuts(): (event: React.KeyboardEvent) => void {
```

Comment:
CodeMirror has its own way of specifying key handlers, so this is now specific to EditAccesskey.

As a follow-up change, we should probably move these handlers out of the editors, so that they're applied also when the focus isn't on the input field, rather like `Alt/Option+ArrowDown/ArrowUp`.